runtime/help/plugins.md

```diff
@@ -52,22 +52,22 @@ as Go's GOOS variable, so `darwin`, `windows`, `linux`, `freebsd`...)
 * `GetOption(name string)`: returns the value of the requested option
 
 * `AddOption(name string, value interface{})`: sets the given option with the given
-   value (`interface{}` means any type in Go).
+   value (`interface{}` means any type in Go)
```

Comment:
I removed the periods here to be consistent with the other list items in this list (sentence fragments lack periods).